lib/versioning/pvp/readme.md

```diff
@@ -0,0 +1,20 @@
+# PVP
+
+**What type of versioning is used?**
+
+> A package version number SHOULD have the form A.B.C, and MAY optionally have any number of additional components, for example 2.1.0.4 (in this case, A=2, B=1, C=0).
+> A.B is known as the major version number, and C the minor version number.
+> Specification: https://pvp.haskell.org/
```

Comment:
```suggestion
Quotes from the [Haskell PVP Specification](https://pvp.haskell.org/):

> A package version number **SHOULD** have the form A.B.C, and **MAY** optionally have any number of additional components, for example 2.1.0.4 (in this case, A=2, B=1, C=0).

> A.B is known as the _major_ version number, and C the _minor_ version number.
```

Changes:

- Copied the style/formatting of the source quotes
- Split the quote into two parts, because the original quotes have some text between them
- Put the link to the document on top, and used a readable link name (important for screenreaders) 😉